### DISCUSSION TITLE
In the local mode of RocketMQ 5, the pushconsumer for a FIFO queue experiences delays while consuming.

### DISCUSSION THREAD

**Discussion by Serajoon** [apache/rocketmq#6607] [Q&A]
In RocketMQ 5, if the pushconsumer for FIFO queue consumes in Local mode, it may experience a phenomenon of consuming for a period of time and then pausing for a while, with a total fixed duration of 30 seconds.
This phenomenon is regular, where the pushconsumer stops after consuming for a certain period of time, and the total duration of consumption and pause time is always 30 seconds.

**Comment by drpmma** [comment]
Please provide more information about your client and your server environment.
